### PR TITLE
Modified tests to use users from json files and security tokens.

### PR DESCRIPTION
**Pull request by NKPmedia**
Before they used the normal login form.

### CODE REVIEW

File: user-test-data/BuvoUser.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "stamm":"Ambronen",
+  "roles":["ROLE_ambrone",
+    "ROLE_nordlichter",
+    "ROLE_wiki",
+    "ROLE_elder",
+    "ROLE_buvo",
+    "ROLE_USER"],
+  "password":"",
+  "salt":""
+}
```
Comment (bone4) [inline user-test-data/BuvoUser.json]: no new line at end of file

File: user-test-data/StavoAmbrone.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "ROLE_wiki",
+    "ROLE_webmaster@schulung.pbnl.de",
+    "ROLE_webmaster@ambronen.pbnl.de",
+    "ROLE_groupWithMailingList",
+    "ROLE_groupWithoutMailingList",
+    "ROLE_elder",
+    "ROLE_USER"],
+  "password":"",
+  "salt":""
+}
```
Comment (bone4) [inline user-test-data/StavoAmbrone.json]: no new line at end of file

File: user-test-data/TestGrueppling.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "username":"TestAmbrone1",
+  "clearPassword":"",
+  "generatedPassword":"",
+  "shaHashedPassword":"",
+  "stamm":"Ambronen",
+  "roles":["ROLE_ambronen",
+    "ROLE_USER"],
+  "password":"",
+  "salt":""
+}
```
Comment (bone4) [inline user-test-data/TestGrueppling.json]: no new line at end of file

File: user-test-data/Tronjer.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "shaHashedPassword":"",
+  "stamm":"Ambronen",
+  "roles":["ROLE_tronjer",
+    "ROLE_nordlichter",
+    "ROLE_wiki",
+    "ROLE_elder",
+    "ROLE_USER"],
+  "password":"",
+  "salt":""
+}
```
Comment (bone4) [inline user-test-data/Tronjer.json]: no new line at end of file

**Review by bone4 — CHANGES_REQUESTED** [review]
sieht bis auf die vier Zeilenende inhaltlich gut aus

File: user-test-data/Tronjer.json (hunk 4 above)
Comment (NKPmedia) [reply user-test-data/Tronjer.json]: Added

File: user-test-data/TestGrueppling.json (hunk 3 above)
Comment (NKPmedia) [reply user-test-data/TestGrueppling.json]: Added

File: user-test-data/StavoAmbrone.json (hunk 2 above)
Comment (NKPmedia) [reply user-test-data/StavoAmbrone.json]: Added

File: user-test-data/BuvoUser.json (hunk 1 above)
Comment (NKPmedia) [reply user-test-data/BuvoUser.json]: Added

**Pull request merged**